apis/nucleus/src/components/listbox/components/grid-list-components/set-overflow-disclaimer.js

```diff
@@ -0,0 +1,40 @@
+const getIndex = (renderProps, isColumnLayout) => {
+  if (renderProps?.visibleRowStopIndex || renderProps.visibleColumnStopIndex) {
```

Comment:
What happens if any of the indices equals `0`?

Reply:
It should never occur since `visibleStopIndex` is the last visible index. But I added a `undefined`-check just in case.